src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+  title = 'projects-index';
```

Comment:
Do we use the title variable?

Reply:
No, it was autogenerated by angular

Reply:
I think it was only being used in the example HTML page.

Reply:
shall I remove it?

Reply:
removed